src/api/video.ts

```diff
@@ -1,38 +1,60 @@
 import Bugsnag from "@bugsnag/js";
 import firebase from "firebase/app";
 
-export interface GetVideoTokenProps {
+export interface GetTwilioVideoTokenProps {
   userId: string;
   roomName: string;
-  onError?: (msg: string) => void;
-  onFinish?: () => void;
```

Comment:
[note] Refactored to remove this legacy pattern. There's no point to this sort of callback when we are using `Promise`. Nothing was using these callbacks in the codebase anyway.